test/test_epm/test_gp.py

```diff
@@ -1,77 +1,238 @@
-import unittest
 import unittest.mock
 
-import george
+import scipy.optimize
 import numpy as np
 import sklearn.datasets
 import sklearn.model_selection
+import skopt.learning.gaussian_process
```

Comment:
Do we use this?

Reply:
Yes, we use it together with the mocking library.